src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -144,7 +145,7 @@ const ModalAction = (props) => {
 
 const styles = StyleSheet.create({
   modalContainer: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `modalContainer` style has been changed from white to almost black. Make sure this change aligns with your application's color scheme and doesn't affect readability or user experience.

```diff
-     backgroundColor: 'white',
+     backgroundColor: COLORS.almostBlack,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->